Azure/sonic-buildimage
# Add build as7716-32xb to platform-modules-accton.mk file

**- What I did**
Add build as7716-32xb to platform-modules-accton.mk
**- How I did it**
Add build as7716-32xb to platform-modules-accton.mk
**- How to verify it**
make target/debs/sonic-platform-accton-as7716-32xb_1.1_amd64.deb-clean
make target/debs/sonic-platform-accton-as7716-32xb_1.1_amd64.deb
Make sure there is no compile error.
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
